servers/zms/src/main/java/com/yahoo/athenz/zms/store/jdbc/JDBCConnection.java

```diff
@@ -3701,6 +3709,48 @@ private void populateDomainRoleMembersMapFromResultSet(Map<String, List<DomainRo
         return targetRoles;
     }
 
+    @Override
+    public boolean processExpiredPendingMembers(int pendingRoleMemberLifespan, String monitorIdentity) {
```

Comment:
The issue with this implementation is that we're not logging these expiry in our official audit log file. We keep track of them in the per-role but those are not permanent - e.g. if I delete the role, the history is gone as well.

So we need to make sure to log these in the audit log file. So the implementation should just fetch the expired pending members from DBService class and then go through each one and execute an expired operation - having the system monitor should be sufficient to identify that this was an expiry operation.